modules/apps/frontend-js/frontend-js-alert-support-web/types/src/main/resources/META-INF/resources/index.d.ts

```diff
@@ -12,10 +12,4 @@
  * details.
```

Comment:
@kresimir-coko sorry, I meant to comment on this file. It looks like you deleted `get_countries.es.js` on accident?

Reply:
Yea, it seems to be the case :wtf: